ansible/roles/edu-sharing/templates/mirror-vocabs.sh.j2

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
 {% for mirror in mirror_vocabularies %}
-wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
-chown -R www-data:www-data {{ mirror.target_directory }}
+    wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
+    chown -R www-data:www-data {{ mirror.target_directory }}
+    {% if ((mirror.isSorted | default("false") == 'true') and (mirror.variant | default("false") == 'true')) %}
```

Comment:
regarding indent of lines in jinja2-templates: you don't need to use indentation for your code, but just for the resulting file. And this file don't need indentation here. So, please remove

Reply:
is ready